pkg/bpftrace/build.yml

```diff
@@ -0,0 +1,13 @@
+image: bpftrace
+network: true
+arches: 
+  - amd64
+gitrepo: github.com/iovisor/bpftrace
```

Comment:
this is not needed.

Reply:
I can remove this. It was part of the packaging readme https://github.com/linuxkit/linuxkit/blob/master/docs/packages.md#package-source so I added.

Reply:
I think this refers to the location of the `build.yml` and defaults to the linuxkit repository so can be omited

Reply:
Gotcha. Fixed in the next commit.